### ISSUE TITLE
Allow theme to define which actor is focusable (via key TAB) and define the order

### ISSUE THREAD

**Issue by gmc-holle** [gmc-holle/xfdashboard#104]
A theme designer should be able to redefine the default tab order and also which actors should be focusable at all.

So the layout XML file should be improved to allow one and only one new child `<focusables>` after `<interface>`. The `<focusables>` tag allows children of tag `<focus>` with an attribute called `id` which refers to a named actor in `<interfaces>`. The order of child tags `<focus>` defines the order at how to move focus within the stage. It doesn't matter if `<focusable>` is define before or after the top-level `<object>` (which is a child of `<interface>`) as it should be handled after the file has been read.

To keep current and/or old theme in a working state the current default behaviour with a fixed list of focusable actors and a fixed order should be applied if `<focusables>` is missing.

The new layout XML file will look like this:

```
<interface>
  <object class="XfdashboardStageInterface" id="primary">
    <child>
      ...
    </child>
  </object>

  <focusable>
    <focus id="searchbox" />
    <focus id="viewpad" />
    <focus id="workspace-selector" />
    <focus id="quicklaunch" />
    ...
  </focusable>
</interface>
```

**Comment by gmc-holle** [comment]
Fully implemented with commits

6da1b4dab37e00c09fd61a8083c14f0d03163679
61969ad776c7f6ad642a61e420319dff337d30d0
e17aee8942c702a703da4f6c5b979cdc786ae76e
b8cdc29081c94cb9aecfe4af4f465c4cd26a7070

and

c6aad4dc1621c4c967d0395ac40860b5757f9987

Closing issue
